docs/RECIPES.md

```diff
@@ -0,0 +1,136 @@
+# Preflight Usage Examples
+
+Below are detailed examples on how to leverage `preflight` in various
+environments.
+
+These examples are shown using the Operator policy against an operator bundle
+(e.g. `preflight check operator <bundle`>), but the same concepts can be applied
+to the Container policy (give or take a few configuration changes).
+
+You will also need an index image containing your bundle for each of these approaches.
+See [DOCS](BUILDING_AN_INDEX.md)
+
+## As a Binary on Your Workstation
+
+To run `preflight` on your workstation, you'll first need to download and
+[install](../README.md#Installation)
```

Comment:
Why is this wrapped weird?

Reply:
Markdown linting was likely complaining about line length. When rendered, they should collapse to a single line.